src/pages/ui/settings.page.ts

```diff
@@ -0,0 +1,210 @@
+  }
+
+  async openProfilFontSizeUp() {
+    await this.profilFontSize.press('ArrowUp');
+  }
+
+  async openProfilFontSizeDown() {
+    await this.profilFontSize.press('ArrowDown');
+  }
+
+  async openProfilSave() {
+    await this.profilSave.click();
+  }
+
+  async openProfilChangePassword() {
+    await this.profilChangePassword.click();
+  }
+
+  async openProfilOldPasswordField() {
+    await this.profilOldPassword.click();
+  }
+
+  async openProfilOldPasswordInput(oldPassword: string) {
+    await this.profilOldPassword.fill(oldPassword);
+  }
+
+  async openProfilNewPasswordField() {
+    await this.profilNewPassword.click();
+  }
+
+  async openProfilNewPasswordInput(newPassword: string) {
+    await this.profilNewPassword.fill(newPassword);
+  }
+
+  async openProfilNewPasswordAgainField() {
+    await this.profilNewPasswordAgain.click();
+  }
+
+  async openProfilNewPasswordAgainInput(newPasswordAgain: string) {
+    await this.profilNewPasswordAgain.fill(newPasswordAgain);
+  }
+
+  async openProfilChangePasswordB() {
+    await this.profilChangePasswordB.click();
+  }
+
+  async openAccessCodeAddSaveDelete(accessCode: string): Promise<SettingsPage> {
+    await this.openProfil();
+    await this.openProfilMAC();
+    await this.openProfilAAC();
+    await this.openProfilACField();
+    await this.accessCodeInput(accessCode as string);
+    await this.openProfilSAAC();
+    await expect(this.page.getByRole('gridcell', { name: accessCode })).toHaveText(accessCode as string);
+    this.page.getByRole('row', { name: 'Available actions ' + accessCode + ' No associated event' }).getByRole('button', { name: 'Available actions' }).locator('svg');
+    await this.openDelete();
+    await this.openClose();
+    return new SettingsPage(this.page);
+  }
+
+  async openAccessCodeUnique(accessCode): Promise<SettingsPage> {
+    await this.openProfil();
+    await this.openProfilMAC();
+    await this.openProfilAAC();
+    await this.openProfilACField();
+    await this.accessCodeInput(accessCode as string);
+    await this.openProfilSAAC();
+    await this.openProfilAAC();
+    await this.openProfilACField();
+    await this.accessCodeInput(accessCode as string);
+    await this.openProfilSAAC();
+    await this.openErrorAC();
+    await this.openSelectAC();
+    await this.openDelete();
+    await this.openClose();
+    return new SettingsPage(this.page);
+  }
+
+  async openSettingsChangeFontSize(): Promise<SettingsPage> {
+    await this.openProfil();
+    await this.openProfilSettings();
+    await this.openFontSizeField();
+    await this.openProfilFontSizeUp();
+    await this.openProfilSave();
+    await this.openProfil();
+    await this.openProfilSettings();
+    await this.openFontSizeField();
+    await this.openProfilFontSizeDown();
+    await this.openProfilSave();
+    return new SettingsPage(this.page);
+  }
+
+  async openPasswordChangeTest(password1, password2): Promise<SettingsPage> {
```

Comment:
It would be more explicit to have password1 renamed to oldPassword and password2 => newPassword